lexicons/place/stream/default/metadata.json

```diff
@@ -0,0 +1,166 @@
+      "properties": {
+        "creator": {
+          "type": "string",
+          "description": "Name of the creator of the work."
+        },
+        "copyrightNotice": {
+          "type": "string",
+          "description": "Copyright notice for the work."
+        },
+        "copyrightYear": {
+          "type": "integer",
+          "description": "Year of creation or publication."
+        },
+        "license": {
+          "type": "string",
+          "description": "License URL or identifier.",
+          "knownValues": [
+            "place.stream.default.metadata#all-rights-reserved",
+            "place.stream.default.metadata#cc0_1__0",
+            "place.stream.default.metadata#cc-by_4_00",
+            "place.stream.default.metadata#cc-by-sa_4__0",
+            "place.stream.default.metadata#cc-by-nc_4__0",
+            "place.stream.default.metadata#cc-by-nc-sa_4__0",
+            "place.stream.default.metadata#cc-by-nd_4__0",
+            "place.stream.default.metadata#cc-by-nc-nd_4__0"
+          ]
+        },
+        "creditLine": {
+          "type": "string",
+          "description": "Credit line for the work."
+        }
+      }
+    },
+    "distributionPolicy": {
+      "type": "object",
+      "description": "Distribution and rebroadcast policy.",
+      "properties": {
+        "allowArchive": {
+          "type": "boolean",
+          "description": "Whether nodes can archive this stream."
+        },
+        "broadcastExpiry": {
+          "type": "string",
+          "format": "datetime",
+          "description": "When rebroadcast permissions expire. If not specified, there is no expiration on rebroadcast permission."
+        }
+      }
+    },
+    "death": {
+      "type": "token",
+      "description": "The content could be perceived as offensive due to the discussion or display of death."
+    },
+    "drugUse": {
+      "type": "token",
+      "description": "The content contains a portrayal of the use or abuse of mind altering substances."
+    },
+    "fantasyViolence": {
+      "type": "token",
+      "description": "The content contains violent actions of a fantasy nature, involving human or non-human characters in situations easily distinguishable from real life."
+    },
+    "flashingLights": {
+      "type": "token",
+      "description": "The content contains flashing lights that could be harmful to viewers with seizure disorders such as photosensitive epilepsy."
+    },
+    "language": {
+      "type": "token",
+      "description": "The content could be perceived as offensive due to the language used."
+    },
+    "nudity": {
+      "type": "token",
+      "description": "The content could be perceived as offensive due to nudity."
+    },
+    "PII": {
+      "type": "token",
+      "description": "The content contains information that can be used to identify a particular individual, such as a name, phone number, email address, physical address, or IP address."
+    },
```

Comment:
I know this is coming right from NewsCodes but I'm not sure there's utility to this one. Presumably  this shouldn't be archived at all, idk.